google/cloud/spanner/samples/samples.cc

```diff
@@ -1628,6 +1628,273 @@ void ListDatabaseOperationsCommand(std::vector<std::string> argv) {
   ListDatabaseOperations(std::move(client), argv[0], argv[1]);
 }
 
+//! [create-backup-schedule] [START spanner_create_backup_schedule]
+void CreateBackupSchedule(
+    google::cloud::spanner_admin::DatabaseAdminClient client,
+    std::string const& project_id, std::string const& instance_id,
+    std::string const& database_id, std::string const& backup_schedule_id) {
+  google::spanner::admin::database::v1::BackupSchedule backup_schedule;
+  *backup_schedule.mutable_full_backup_spec() = {};
+  backup_schedule.mutable_spec()->mutable_cron_spec()->set_text("0 0 * * *");
+  backup_schedule.mutable_retention_duration()->set_seconds(3600 * 24 * 7);
+
+  google::cloud::spanner::Database db(project_id, instance_id, database_id);
+  google::spanner::admin::database::v1::CreateBackupScheduleRequest request;
+  request.set_parent(db.FullName());
+  request.set_backup_schedule_id(backup_schedule_id);
+  *request.mutable_backup_schedule() = std::move(backup_schedule);
+
+  auto created_backup_schedule = client.CreateBackupSchedule(request);
+  if (!created_backup_schedule)
+    throw std::move(created_backup_schedule).status();
```

Comment:
style nit: we use braces `{ }` for if statements that do not fit on a single line.

Reply:
Done.